src/main/java/com/templateproject/api/entity/Article.java

```diff
@@ -48,20 +45,27 @@ public class Article {
     @Transient
     private String categorySlug;
 
+    //gestion des commentaires 
     @OneToMany(mappedBy = "article")
     @JsonManagedReference
     @JsonIgnore
     private List<Comment> listComments;
+    
+    //gestion des paragraphes
+    @OneToMany(mappedBy = "article")
+    //@JsonManagedReference
+    //@JsonIgnore
+    private List<Paragraphe> listParags;
```

Comment:
Be careful with your variables names.It should be explicit and in english : Paragraph instead of Paragraphe; listParagraphs instead of listParags